### PR TITLE
=str #16652 Add java docs for 'Buffers and rate'

### PR DESCRIPTION
**Pull request by patriknw**
Refs #16652

### CODE REVIEW

File: akka-docs-dev/rst/java/stream-rate.rst [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+waiting job. This is the preferred strategy if jobs are expected to be
+resent if not processed in a certain period. The oldest element will be
+retransmitted soon, (in fact a retransmitted duplicate might be already in the queue!)
+so it makes sense to drop it first.
+
+.. includecode:: ../../../akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/StreamBuffersRateDocTest.java#explicit-buffers-drophead
+
+Compared to the dropping strategies above, dropBuffer drops all the 1000
+jobs it has enqueued once the buffer gets full. This aggressive strategy
+is useful when dropped jobs are preferred to delayed jobs.
```
Comment (rkuhn) [inline akka-docs-dev/rst/java/stream-rate.rst]: “… when _dropping_ jobs _is_ preferred to _delaying_ jobs.”

**Comment by rkuhn** [comment]
LGTM

**Comment by ktoso** [comment]
LGTM

File: akka-docs-dev/rst/java/stream-rate.rst (hunk 1 above)
Comment (patriknw) [inline akka-docs-dev/rst/java/stream-rate.rst]: thanks, fixed

**Pull request merged**